packages/storybook/src/nimble/text-field/text-field-matrix.stories.ts

```diff
@@ -492,6 +519,16 @@ export const colorThemeReadOnlyDisabledWithButtons: StoryFn = createFixedThemeSt
     colorThemeDarkGreenBackground
 );
 
+export const textFieldRequiredVisibleThemeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrix(requiredVisibleStatesComponent, [
+        requiredVisibleStates,
```

Comment:
nit: could you move uuup the appearanceStates to the second dimension? So the grouping is first split by requiredVisible and then by appearance mode? Think it's easier to visually scan when the same appearance modes are clustered together